gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStreamIntegrationTest.java

```diff
@@ -267,21 +267,17 @@ public void operation_durationMetric_tests() throws Exception {
     assertThat(in.read(1, value, 1, 1)).isEqualTo(1);
     assertThat(value).isEqualTo(expected);
 
-    // One seek is to seek to the position and another seek to seek back to the original position.
-    TestUtils.verifyCounter(stats, STREAM_READ_SEEK_OPERATIONS, 2);
+    TestUtils.verifyCounter(stats, STREAM_READ_SEEK_OPERATIONS, 0);
     TestUtils.verifyCounter(stats, STREAM_READ_SEEK_BACKWARD_OPERATIONS, 2);
 
-    TestUtils.verifyDurationMetric(stats, STREAM_READ_OPERATIONS, 2);
+    TestUtils.verifyDurationMetric(stats, STREAM_READ_OPERATIONS, 0);
```

Comment:
instead of verifying individual metric to be `0` what about verifying that nothing will be captured unless stream is closed?

Reply:
some metrics will still be captured before close. Like backward/forward seek.